CONTRIBUTING.md

```diff
@@ -41,15 +41,15 @@ To run tests:
 - Clone the repository:
 
   ```console
-  $ git clone git@github.com:zendframework/zend-expressive-fastroute.git
+  $ git clone git://github.com/zendframework/zend-expressive-fastroute.git
   $ cd zend-expressive-fastroute
   ```
 
 - Install dependencies via composer:
 
   ```console
   $ curl -sS https://getcomposer.org/installer | php --
-  $ ./composer.phar install
+  $ composer install
```

Comment:
This doesn't work. `curl … | php --` just downloads `composer.phar`, but does not install composer globally – Same issue as the one already mentioned at https://github.com/zendframework/zend-stratigility/pull/97#commitcomment-20593201 :unamused: